src/ServiceManagement/Compute/ComputeManagement/ComputeManagement.csproj

```diff
@@ -32,9 +33,18 @@
     <Reference Include="Microsoft.Azure.Common">
       <HintPath>$(LibraryNugetPackageFolder)\Microsoft.Azure.Common.2.0.4\lib\portable-net45+wp8+wpa81+win\Microsoft.Azure.Common.dll</HintPath>
     </Reference>
-    <Reference Include="System.Threading.Tasks" />
+    <Reference Include="Newtonsoft.Json, Version=6.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
```

Comment:
You should not need to make changes to this csproj, other than including any new generated files